shopinvader_product_binding/__manifest__.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Shopinvader_v1_product",
+    "summary": "Shopinvader Product",
+    "version": "14.0.5.24.12",
+    "category": "e-commerce",
+    "website": "https://github.com/shopinvader/odoo-shopinvader",
+    "author": "Akretion",
+    "license": "AGPL-3",
+    "application": True,
+    "installable": True,
+    "external_dependencies": {"python": ["cerberus", "unidecode"], "bin": []},
+    "depends": [
+        "shopinvader_restapi",
+        # "component_event",  # product_event_listener
+        # "base_url",  # shopinvader.category
+        # "base_sparse_field_list_support",  # shopinvader.binding
+        "base_sparse_field",
+        "base_rest",
+        "jsonifier",
+        "base_sparse_field_list_support",
+        "base_url",
+        "base_vat",
+        "component_event",
+        "sale",
+        "sale_discount_display_amount",
+        "server_environment",
+        "onchange_helper",
+        "queue_job",
+        "mail",
+    ],
+    "data": [
+        "security/ir.model.access.csv",
+        "security/shopinvader_category_security.xml",
+        "security/shopinvader_product_security.xml",
+        "security/shopinvader_variant_security.xml",
+        "wizards/shopinvader_variant_unbinding_wizard.xml",
+        "wizards/shopinvader_variant_binding_wizard.xml",
+        "wizards/shopinvader_category_binding_wizard.xml",
+        "wizards/shopinvader_category_unbinding_wizard.xml",
+        "views/shopinvader_menu.xml",
+        "views/shopinvader_product_view.xml",
+        "views/shopinvader_variant_view.xml",
+        "views/shopinvader_category_view.xml",
+        "views/shopinvader_backend_view.xml",
```

Comment:
why moving shopinvader backend view ?

Reply:
The shopinvader menu view has been split, with only the menu linked to the product in the product_binding module.
[https://github.com/akretion/odoo-shopinvader/blob/14.0-split_shopinvader_v1_product/shopinvader_restapi/views/shopinvader_menu.xml](url)

However, the shopinvader backend view hasn't been split as it does, so I'll take care of that.

Reply:
Hi, the more I digg into it, the more I doubt it's the good solution.
We need one basic shoinvader module (with basic menus), shopinvader_restapi should depend of it

Reply:
According to Seb, the menu have to stay in shopinvader_restapisecurity, 
ir_export product and category stays in shopinvader_product_binding